src/sage/schemes/elliptic_curves/weierstrass_morphism.py

```diff
@@ -973,6 +973,105 @@ def inseparable_degree(self):
         """
         return Integer(1)
 
+    def is_identity(self):
+        r"""
+        Returns if the Weierstrass isomorphims is the identity.
+
+        EXAMPLES::
+
+            sage: from sage.schemes.elliptic_curves.weierstrass_morphism import *
+            sage: p = 97
+            sage: Fp = GF(p)
+            sage: E = EllipticCurve(Fp, [1, 28])
+            sage: ws = WeierstrassIsomorphism(E, None, E)
+            sage: ws.is_identity()
+            False
+
+        ::
+
+            sage: from sage.schemes.elliptic_curves.weierstrass_morphism import *
+            sage: p = 97
+            sage: Fp = GF(p)
+            sage: E = EllipticCurve(Fp, [1, 28])
+            sage: ws = WeierstrassIsomorphism(E, (1, 0, 0, 0), None)
+            sage: ws.is_identity()
+            True
+        """
+        return self.tuple() == (1, 0, 0, 0)
+
+    def order(self):
+        r"""
+        If the Weierstrass isomorphism is an automorphism, computes the order of the automorphism.
```

Comment:
As above, it should start with a verb. Something like "Compute the order of this Weierstrass isomorphism if it is an automorphism. A `ValueError` is raised otherwise."